lib/index.ts

```diff
@@ -0,0 +1,20 @@
+import { Authorizer } from './authorizer';
+import { AuthorizerOptions, ExpressRequest } from './interfaces';
+
+export default function(options: AuthorizerOptions) {
+  const authorizer = new Authorizer(options);
+
+  const middleware = function(request: ExpressRequest, response: any, next) {
+    authorizer.getFromAuthorizationHeader(request);
+
+    const auth = request.logentriesWebhookAuth || {};
+
+    if (auth.user === null) {
```

Comment:
It is customary in expressjs for an object representing the authenticated user to be put into the `user` property of the `request` object.  This for instance is where passportjs puts an object representing the user, and is typical in sample client code using information about an user authenticated by expressjs middleware.

While I don't care much for mutating the state of the request object in the getter function, I think it's wholly appropriate and typical to expect middleware functions to mutate the state of the request object.